node/src/lib/genetal-utils.js

```diff
@@ -1,8 +1,17 @@
-const convertRequiresApprovalToBoolean = str => {
+const convertStringToBoolean = str => {
   if (str === 'true') return true;
   if (str === 'false') return false;
 };
 
+const removeEmptyValuesFromObj = payload =>
+  JSON.parse(
+    JSON.stringify(payload, (_, value) => {
+      if (value === undefined) return undefined;
```

Comment:
this function always returns `value` :)
You can `JSON.parse(JSON.stringify(payload));`